packages/react-components/package.json

```diff
@@ -39,12 +39,14 @@
     "@fluentui/babel-make-styles": "9.0.0-beta.4"
   },
   "dependencies": {
+    "@fluentui/react": "8.49.0",
     "@fluentui/react-accordion": "9.0.0-beta.5",
     "@fluentui/react-avatar": "9.0.0-beta.4",
     "@fluentui/react-badge": "9.0.0-beta.4",
     "@fluentui/react-button": "9.0.0-beta.5",
     "@fluentui/react-card": "9.0.0-beta.5",
     "@fluentui/react-divider": "9.0.0-beta.4",
+    "@fluentui/react-icons": "2.0.154-beta.5",
```

Comment:
You don't need to add this--it shouldn't be in `dependencies` since it's not referenced by published files, and it will be picked up from the repo root `devDependencies`.

If you're getting an eslint error when referencing the icons via devDependencies, there's an eslint setting you can edit (I'll have to look up what it is)

Reply:
Actually for eslint you'll need to just disable the rule per-line for now (I'll follow up with other people about getting this fixed properly)
```
// eslint-disable-next-line import/no-extraneous-dependencies -- this file can use devDependencies
```